src/restic/node.go

```diff
@@ -162,6 +170,26 @@ func (node Node) restoreMetadata(path string) error {
 		}
 	}
 
+	switch runtime.GOOS {
+	case "windows", "openbsd":
```

Comment:
Why the switch here? For the Node stuff, we're using different files to implement functions differently based on the OS (you've already discovered that), so you can just call `restoreAcls()` even on Windows and this switch is not needed (because `Setxattr()` for Windows is just a no-op).